server.js

```diff
@@ -1,7 +1,39 @@
-const express = require('express')
-const app = express()
-const PORT = 5000
+const express = require('express');
+const mongoose = require('mongoose');
+require('dotenv').config();
+const app = express();
+const PORT = 5000;
+const cors = require('cors');
 
+//Middleware
+app.use(express.json());
+app.use(cors());
+
+//Start server
 app.listen(PORT, () => {
-    console.log(`Backend-Server started successfully at port: ${PORT}`)
-})
+  console.log(`Backend-Server started successfully at port: ${PORT}`);
+});
+
+//DB
+const dbUser = process.env.DB_USER;
+const dbPass = process.env.DB_PASS;
+
+const dbConnect = `mongodb+srv://${dbUser}:${dbPass}@cluster0.1cqrd.mongodb.net/records_db?retryWrites=true&w=majority`;
+
+mongoose
+  .connect(dbConnect, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useFindAndModify: false,
+    useCreateIndex: true,
+  })
```

Comment:
Could we put these options in an object separately maybe `const mongooseOptions = {...}`, it would look cleaner and easier to modify in case we need it. What do you think?

Reply:
it looks cleaner. done.